analytics-datastore-clickhouse/importer/config/clickhouseConfig.js

```diff
@@ -20,7 +20,7 @@ const clickhouse = new ClickHouse({
     try {
       const r = await clickhouse.query(query).toPromise();
 
-      if (r.includes('Exception')) throw new Error(r);
+      if (typeof r === 'string' && r.includes('Exception')) throw new Error(r);
```

Comment:
@nour-borgi Could you check why the result does not match the regex defined in 'clickhouse' package : https://github.com/TimonKK/clickhouse/commit/a84d890a1d2d509f1377b1c9abf7da28307a42fc

Reply:
Amazing! 🚀  done in [e86e3e2](https://github.com/jembi/platform/pull/188/commits/e86e3e29d8caff3771838c7ec50f3bbb4b9004e0)